apps/web/src/views/NFTs/NFTCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Card, CardBody, Center, Heading, Image } from "@chakra-ui/react";
+import { type NFTBalance, thumbnailUri } from "@umami/core";
+import { getIPFSurl } from "@umami/tezos";
+
+import { useColor } from "../../styles/useColor";
+
+export const NFTCard = ({ nft }: { nft: NFTBalance }) => {
+  const color = useColor();
+  const url = getIPFSurl(thumbnailUri(nft));
+  const fallbackUrl = getIPFSurl(nft.displayUri);
+  const name = nft.metadata.name;
+
+  return (
+    <Card
```

Comment:
Hover is missing

<img width="579" alt="image" src="https://github.com/user-attachments/assets/81d0b2bd-370e-4ecd-a225-9d865aeee903">